packages/zed/zed.2.0.2/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "opam-devel@lists.ocaml.org"
+authors: ["Jérémie Dimino"]
+homepage: "https://github.com/ocaml-community/zed"
+bug-reports: "https://github.com/ocaml-community/zed/issues"
+dev-repo: "git://github.com/ocaml-community/zed.git"
+license: "BSD3"
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "dune" {build & >= "1.0.0"}
+  "base-bytes"
+  "camomile" {>= "1.0.1"}
+  "react"
+  "charInfo_width" {>= "1.1.0" & < "2.0~"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
```suggestion
```
You don't seem to use dune substitutions anywhere. Is there a reason why you are using `dune subst` here?

Reply:
no, this is a good time to clean up some old useless code.
Thank you.